.github/workflows/test-suite.yml

```
@@ -0,0 +1,64 @@
+name: Test Suite
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+
+jobs:
+  Code-Climate-Tests:
+    runs-on: ubuntu-latest
+
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_DATABASE: sessiontome
+          MYSQL_USER: application
+          MYSQL_PASSWORD: secret
+          MYSQL_ROOT_PASSWORD: secretroot
+        ports:
+          - 3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.composer/cache/files
+          key: dependencies-laravel-6.x-php-7.4-composer-${{ hashFiles('composer.json') }}
```

We should consider the following changes to the label.
```suggestion
          key: dependencies-laravel-8.x-php-8.0-composer-${{ hashFiles('composer.json') }}
```

---

Ah whoops missed that!